models/ReportContent.php

```diff
@@ -117,12 +117,12 @@ public static function canReportPost(ContentActiveRecord $post, $userId = null)
         $user = ($userId != null) ? User::findOne(['id' => $userId]) : Yii::$app->user->getIdentity();
 
         if ($user == null || $user->super_admin) {
```

Comment:
I think this was meant, that super users do not have to report content, since they can remove them if its not appropriate.